src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,40 @@
+package core.basesyntax;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class Main {
+    public static final Map<String, Integer> shop = new HashMap<>();
+
+    public static void main(String[] arg) throws IOException {
+        // 1. Read the data from the input CSV file
+        List<String> inputReport = Files.readAllLines(Path.of("C:/txt/reportToRead.csv"));
```

Comment:
Extract logic to separate service. So when you have additional source of data you can just write new service implementation